features/topology-map/plugins/org.opennms.features.topology.plugins.topo.application/src/main/java/org/opennms/features/topology/plugins/topo/application/ApplicationSearchProvider.java

```diff
@@ -92,8 +92,9 @@ public boolean supportsPrefix(String searchPrefix) {
         bldr.limit(10);
         Criteria dbQueryCriteria = bldr.toCriteria();
 
+        // TODO: patrick do we want to replace the dao call also with a call to GraphService?
```

Comment:
You are right, at some point we want to call the `GraphService` for this calculation.
To implement this, we need search capabilities on the `GraphService` side, so we should implement that next.